ios/engine/KMEI/KeymanEngine/Classes/KeymanWebViewController.swift

```diff
@@ -399,6 +412,31 @@ extension KeymanWebViewController: WKScriptMessageHandler {
     } else if fragment.hasPrefix("#beep-") {
       beep(self)
       delegate?.beep(self)
+    } else if fragment.hasPrefix("#suggestPopup"){
+      let baseFrameKey = fragment.range(of: "+baseFrame=")!
```

Comment:
One thing - iOS isn't using a URL here.  It may be using the same syntax (with `var hash` and all), but it's actually passing a plain `String` across the barrier.  (Does mean we don't have to worry about decoding JSON from a URL-based encoding, at least.)